games-simulation/openttd/openttd-12.2.recipe

```diff
@@ -29,6 +29,7 @@ REQUIRES="
 	lib:libpng16$secondaryArchSuffix
 	lib:libsdl$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
```

Comment:
A few changes, first one is a leftover probably (cosmetic):
```diff
-	lib:libsdl$secondaryArchSuffix
+	lib:libSDL_1.2$secondaryArchSuffix
	lib:libz$secondaryArchSuffix
-	lib:libgl$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
```
For libSDL also in devel:
Sort alphabetically